.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] [REQUIRED] Good job so far! There should be a `.gitignore` file listing `test.md` file directory 🙏🏼 

![image](https://github.com/migcm06/Hello-Microverse/assets/63915024/5f72c6f8-73e5-4346-8ee8-7e4b954d7771)